invenio/modules/records/upgrades/records_2015_05_06_recjson_history_table.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTIiULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+
+import warnings
+import sqlalchemy as sa
+from invenio.ext.sqlalchemy import db
+from invenio.modules.upgrader.api import op
+
+
+depends_on = []
+
+
+def info():
+    return "Create table history_record_json."
+
+
+def do_upgrade():
+    """Create new table for the record history.
+
+    To fill it up please use:
+    `python scripts/history_record_migration_kit.py run`.
+    """
+    if not op.has_table("record_json_history"):
+        op.create_table(
+            'history_record_json',
+            sa.Column(
+                'id', db.MediumInteger(8, unsigned=True), nullable=False),
+            sa.Column('revision', db.DateTime, nullable=False),
+            sa.Column('json', db.JSON, nullable=False),
+            sa.PrimaryKeyConstraint('id', 'revision'),
+            mysql_charset='utf8',
+            mysql_engine='MyISAM'
```

Comment:
We can discuss about MyISAM vs InnoDB, but I think the point here is that I copy paste from other upgrade recipe, see `git grep 'op\.create_table'`

I'm not sure whether is necessary or not to specify here the engine, anyhow this is the one used by [default](https://github.com/inveniosoftware/invenio/blob/master/invenio/ext/sqlalchemy/__init__.py#L142) in Invenio.

cc @lnielsen